CRM/Contact/DAO/SavedSearch.php

```diff
@@ -189,7 +189,7 @@ public static function &fields() {
         'name' => [
           'name' => 'name',
           'type' => CRM_Utils_Type::T_STRING,
-          'title' => ts('Saved Search Name'),
+          'title' => ts('Name'),
```

Comment:
I thought where we landed was that the title would assume low-context and `<html<label>` would assume high context - so instead of reducing the information in the title we would add a html label tag